src/main/java/seedu/address/model/util/InternshipItemUtil.java

```diff
@@ -0,0 +1,13 @@
+package seedu.address.model.util;
+
+/**
+ * InternshipItemUtil class which provides the strings used in the InternshipItem.
+ */
+public class InternshipItemUtil {
+
+    public static final String COMPANY_OUTPUT_NAME = " Company Name: ";
+    public static final String PERIOD_OUTPUT_NAME = " Company Name: ";
+    public static final String WAGE_OUTPUT_NAME = " Wage: ";
```

Comment:
Sorry may I know what's the difference between `COMPANY_OUTPUT_NAME` and `PERIOD_OUTPUT_NAME`
could they just be summarized into`COMPANY_NAME = "Company Name: "`

Reply:
Good catch, missed this completely